### PR TITLE
Arabic Translation.

### PR DESCRIPTION
**Pull request by ContractISover**
I have made some Arabic translations for this project and I would like to upstream it.

### CODE REVIEW

**Comment by maoschanz** [comment]
thanks!

File: po/ar.po [hunk 1]
```diff
@@ -0,0 +1,2520 @@
...
+
+#: src/main.py
+msgid "Report bugs or ideas"
+msgstr "الإبلاغ عن الأخطاء أو الأفكار"
+
+#. To tranlators: "translate" this by a list of your names (one name
+#. per line), they will be displayed in the "about" dialog
+#: src/main.py
+msgid "translator-credits"
+msgstr "المترجمين"
```
Comment (maoschanz) [inline po/ar.po:1058]: i just noticed this. As suggested by the comment, it was supposed to be the name under which you want to be credited

i'm putting your github account name, but if you want something else please tell me

Comment (ContractISover) [reply po/ar.po:1058]: I would prefer my Codeberg account. https://codeberg.org/contractover
but if you prefer to put my GitHub, that's fine too.

Comment (maoschanz) [reply po/ar.po:1058]: it's a name or a pseudonym, not a link to an actual account, for now i've written "@contractisover"

**Pull request merged**